tests/FakeItEasy.IntegrationTests.netstd/project.json

```diff
@@ -0,0 +1,66 @@
+﻿{
+  "name": "FakeItEasy.IntegrationTests",
+  "version": "1.0.0-*",
+  "description": "FakeItEasy.IntegrationTests.netstd Class Library",
+  "authors": [ "" ],
+  "packOptions": {
+    "tags": [ "" ],
+    "projectUrl": "",
+    "licenseUrl": ""
+  },
+
+  "buildOptions": {
+    "keyFile": "../../FakeItEasy.snk",
+    "compile": {
+      "include": [
+        "../FakeItEasy.IntegrationTests/**/*.cs"
+      ],
+      "exclude": [
+      ]
+    },
+    "define": [ "NETCORE" ]
+  },
+
+  "dependencies": {
+    "xunit.abstractions": "2.0.1-rc2",
+    "xunit.extensibility.core": "2.2.0-beta2-build3300",
+    "xunit.extensibility.execution": "2.2.0-beta2-build3300"
+  },
+
+  "testRunner": "xunit",
+  "frameworks": {
+    "netcoreapp1.0": {
+      "buildOptions": {
+        "define": [ "NETCORE", "FEATURE_NETCORE_REFLECTION" ]
+      },
+      "dependencies": {
+        "Castle.Core": "4.0.0-beta001",
+        "FakeItEasy": {
+          "target": "project",
+          "version": "99.99.99-wrapped"
+        },
+        "FakeItEasy.Tests.netstd": {
+          "target": "project",
+          "version": "1.0.0"
+        },
+        "FluentAssertions": "4.6.3",
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.0.0"
+        },
+        "System.Console": "4.0.0",
+        "System.Diagnostics.TraceSource": "4.0.0",
+        "System.Linq.Expressions": "4.1.0",
+        "System.Reflection": "4.1.0",
+        "System.Reflection.Emit.ILGeneration": "4.0.1",
+        "System.Reflection.Extensions": "4.0.1",
+        "System.Reflection.TypeExtensions": "4.1.0",
+        "System.Runtime.Serialization.Primitives": "4.1.1",
+        "System.Threading.Tasks.Parallel": "4.0.1",
+        "Microsoft.CSharp": "4.0.1",
+        "dotnet-test-xunit": "2.2.0-preview2-build1029"
+      },
+      "imports": [ "dnxcore50", "portable-net451+win81" ]

```

Comment:
OK, I think I understand. So it means that, because dnxcore50 and portable-net451+win81 happen to be compatible with netcoreapp1.0, even though those platforms do not explicitly support a version of netstandard which netcoreapp1.0 supports, the project can still use libraries which target dnxcore50 and portable-net451+win81?